backend/app/api/v1/endpoints/cpt/maps/rhoai.py

```diff
@@ -0,0 +1,26 @@
+from ....commons.rhoai import getData
+from datetime import date
+
+
+################################################################
+# This will return a DataFrame required by the CPT endpoint
+################################################################
+
+async def rhoaiNotebooksPerformanceMapper(start_datetime: date, end_datetime: date, configpath: str):
+    df = await getData(start_datetime, end_datetime, configpath)
+    df.insert(len(df.columns), "product", "RHOAI Notebooks Performance")
+    df.insert(len(df.columns), "ciSystem", "Middleware Jenkins")
+    df.insert(len(df.columns), "releaseStream", "rc")
+    df.insert(len(df.columns), "buildUrl", "N/A")
+    df.insert(len(df.columns), "jobStatus", "success")
+
+    if df.empty:
+        return df
+
+    df["startDate"] = df["metadata.start"]
```

Comment:
can we remove these `metadata.*` columns from the df once they are assigned to new columns as they are not being used further anyway?